Chapter1/4-Shaders-Uniforms/Shaders/shader.frag

```diff
@@ -4,7 +4,8 @@ out vec4 outputColor;
 
 
 // The Uniform keyword allows you to access a shader variable at any stage
-// Of the shader chain. Whatever you set this variable to it keeps it 
+// Of the shader chain as well as being accessible across all of the main the main program.
```

Comment:
This sentence has some broken grammar in the end.

Also super minor.
Only capitalize the first letter of a comment when the previous comment ends with a period.